src/main/java/com/fundation/webservice/controller/Controller.java

```diff
@@ -198,28 +202,28 @@ public PdfResponse pdfToImage(@RequestParam("asset") MultipartFile asset, @Reque
             e.printStackTrace();
         }
         if (inputJson.getString("checksum").equals(inputChecksumString)) {
-        CriteriaPdfToImage criterion = new CriteriaPdfToImage();
-        criterion.setSrcPath(properties.getProperty("file.uploadDir") + pdfName);
-        //criterion.setDestPath(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "\\");
+            CriteriaPdfToImage criterion = new CriteriaPdfToImage();
+            criterion.setSrcPath(properties.getProperty("file.uploadDir") + pdfName);
+            //criterion.setDestPath(properties.getProperty("file.downloadDir") + outputJson.getString("name") + "\\");
             criterion.setDestPath(inputJson.getString("destPath") + outputJson.getString("name") + "\\");
-        criterion.setName(outputJson.getString("name"));
-        criterion.setDpi(new Integer(configJson.getString("dpi")));
-        criterion.setExt(outputJson.getString("ext"));
-        criterion.setFormatColor(configJson.getString("formatColor"));
-        ConvertPdfToImage pdfDocument = new ConvertPdfToImage();
-        pdfDocument.convert(criterion);
-        /**
-         * This line compresses the folder with images in a zip file
-         */
-        FolderZipped.zipFolder(outputJson.getString("name"));
+            criterion.setName(outputJson.getString("name"));
+            criterion.setDpi(new Integer(configJson.getString("dpi")));
+            criterion.setExt(outputJson.getString("ext"));
+            criterion.setFormatColor(configJson.getString("formatColor"));
+            ConvertPdfToImage pdfDocument = new ConvertPdfToImage();
+            pdfDocument.convert(criterion);
+            /**
+             * This line compresses the folder with images in a zip file
```

Comment:
modified the format on this comment to simple comment with an only double backslash "//" and not with documentation format "/**
                                        *
                                        */"

Reply:
The comment is deleted because I considerate, not necessary.